proto/panacea/datadeal/v2alpha2/genesis.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package panacea.datadeal.v2alpha2;
+
+option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+
+import "gogoproto/gogo.proto";
+
+// GenesisState defines the datadeal module's genesis state.
+message GenesisState {
+}
+
+// Params defines the datatdeal module's params.
+message Params {
+  string oracle_commission = 1 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
```

Comment:
You can use `types.Dec` instead of `types.Int` as below, if you intended to accept values which are `0 <= x <= 1`.
https://github.com/cosmos/cosmos-sdk/blob/ec62d939376ec5a46a044615b85339216f47bd56/proto/cosmos/staking/v1beta1/staking.proto#L31

Reply:
I think that the `commission` field should be rate, not the amount of coin. I'll change it.